onboarding.adoc

```diff
@@ -1,327 +1,86 @@
+Whenever people join a new community, discussions on communication channels such as mailing lists or Slack will typically be overwhelming to follow. To alleviate this, it's helpful to carve out a space for new community members as they get started. For example, dedicated channels such as "getting started" or "introductions" can be good places for people to feel safe asking newcomer questions. For more experienced community members, by helping answer questions or even simply offering encouragement on these channels will go a long way in making new community members feel welcome as they're getting started.  
 
-Giving community members a chance to grow within a project benefits those involved.
-Clearing the path to such opportunities is a great way to keep your project healthy and thriving.
+Once new community members feel oriented and comfortable enough to start contributing to an open source project, their next questions will be "where can I contribute?" To help newcomers get started, it's helpful to have a list of work items that people can tackle. Many communities have labels such as "good for first time contributors", "good first issue", "help wanted", etc. that people can use to narrow down the list of issues in their communities. Issues with these labels could range from documentation errors, easy bug fixes, or other simple tasks that new contributors can tackle so that they can have early successes. It's also helpful have a contact person (or people) who can serve as mentors or coaches listed on these issues in case people need help getting started. Although issues may look simple for experienced community members, that may not always be the case for many newcomers. 
 
-== Responding to new contributors
+Many open source communities organize events that community members participate in. Whether the events are in person or virtual, these events provide great opportunities for community members to collaborate synchronously and also to get to know each other in more relaxed settings. These events could be summits, hackathons, user confernces, etc. and it makes sense to create separate sessions (or even a track) for new community members. You could organize formal orientation sessions as a Day 0 event if your budget allows for it, but even a lunch time session where new comers can meet other community members so they know who they can ask questions to later on can go a long way in making community members feel welcome.
 
-Speed is a critical factor in growing your contributor base.
-The moment a new contributor submits a pull request or draft patch, a timer starts.
-And the longer maintainers take to respond to their submission, the lower the chance that person will ever contribute to the project again.
-If you're running a young project and looking to attract lots of new contributors, there may be no higher priority than dealing with first-time contributions quickly.
+== Contributor pathways
```

Comment:
Added some notes for potential section on contributor pathways. I think this is an important facet of community onboarding.